<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="DONTV4-OS36" name="Maven.POM.AppTest" tests="4" failures="0" timestamp="06 Jun 2017 10:17:01 GMT" time="4.256" errors="1">
  <testcase name="testParameterWithXML" time="2.147" classname="Maven.POM.AppTest"/>
  <testcase name="setup" time="0.004" classname="Maven.POM.AppTest">
    <error type="java.lang.IllegalStateException" message="The driver executable does not exist: D:\Java_Huong\selenium\MicrosoftWebDriver.exe">
      <![CDATA[java.lang.IllegalStateException: The driver executable does not exist: D:\Java_Huong\selenium\MicrosoftWebDriver.exe
	at com.google.common.base.Preconditions.checkState(Preconditions.java:518)
	at org.openqa.selenium.remote.service.DriverService.checkExecutable(DriverService.java:136)
	at org.openqa.selenium.remote.service.DriverService.findExecutable(DriverService.java:131)
	at org.openqa.selenium.edge.EdgeDriverService.access$000(EdgeDriverService.java:33)
	at org.openqa.selenium.edge.EdgeDriverService$Builder.findDefaultExecutable(EdgeDriverService.java:63)
	at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:330)
	at org.openqa.selenium.edge.EdgeDriverService.createDefaultService(EdgeDriverService.java:55)
	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:96)
	at Maven.POM.AppTest.setup(AppTest.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.TestRunner.beforeRun(TestRunner.java:626)
	at org.testng.TestRunner.run(TestRunner.java:594)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- setup -->
  <testcase name="testParameterWithXML" time="0.000" classname="Maven.POM.AppTest">
    <skipped/>
  </testcase> <!-- testParameterWithXML -->
  <testcase name="testParameterWithXML" time="2.105" classname="Maven.POM.AppTest"/>
</testsuite> <!-- Maven.POM.AppTest -->
